package.json

```diff
@@ -56,10 +56,15 @@
     "debug-break": "node --nolazy --inspect-brk scripts/kibana --dev",
     "precommit": "node scripts/precommit_hook",
     "karma": "karma start",
-    "lint": "echo 'use `node scripts/eslint` and/or `node scripts/tslint`' && false",
-    "lintroller": "echo 'use `node scripts/eslint --fix` and/or `node scripts/tslint --fix`' && false",
-    "makelogs": "echo 'use `node scripts/makelogs`' && false",
-    "mocha": "echo 'use `node scripts/mocha`' && false",
+    "lint": "node scripts/eslint && node scripts/tslint && node scripts/sasslint",
```

Comment:
Thoughts on instead calling the other yarn script from here? 

`yarn run lint:es && yarn run run lint:ts && yarn run run lint:sass`